src/control/destination/endpoint.rs

```diff
@@ -7,7 +7,7 @@ use conditional::Conditional;
 
 /// An individual traffic target.
 ///
-/// Equality, Ordering, and hashability is determined soley by the Endpoint's address.
+/// Equality, Ordering, and hashability is determined solely by the Endpoint's address.
```

Comment:
I believe this comment is no longer the case, and should actually probably just be removed entirely --- the `Eq`/`PartialEq` implementations are derived now (so they also consider the endpoint's metadata), and the `Hash` and `Ord` implementations have been removed entirely.